test_conformance/relationals/test_shuffles.cpp

```diff
@@ -201,14 +201,13 @@ const char *get_order_string( ShuffleOrder &order, size_t vecSize, cl_uint lengt
 
     size_t j, idx;
 
-    // Assume we don't have to use numbers
-    byNumber = 0;
-    // Check to see
+    // Assume we don't have to use numbered indices (.s0123...).
+    byNumber = false;
```

Comment:
Can we remove the `byNumber` pass-by-value parameter of `get_order_string` if we always set the value to `false` at the start of the function? And just have `bool get_order_string = false;` here instead.